src/Microsoft.Identity.Client/Platforms/Android/BrowserTabActivity.cs

```diff
@@ -31,11 +31,11 @@
 using Android.OS;
 using Microsoft.Identity.Client.Platforms.Android.SystemWebview;
 
-namespace Microsoft.Identity.Client
+namespace Microsoft.Identity.Client.Platforms.Android
```

Comment:
this has to be Microsoft.Identity.Client....unless we're okay w/adding it to the list of breaking changes when we release config object. We'll need to make note of the change for developers.